lib/Models/SensorObservationServiceCatalogItem.js

```diff
@@ -626,18 +633,23 @@ SensorObservationServiceCatalogItem.prototype.showOnSeparateMap = function(globe
     }
 };
 
+/**
+ * Performs the GetFeatureOfInterest request to obtain the locations of sources of data that match the required 
```

Comment:
Don't use jsdoc comments for private functions, because then they'll show up in the reference docs even though they're inaccessible. You can use regular comments. Or if you really want to use jsdoc comments, add `@private`.